frontend/src/pages/workspace/[slug]/1-project-team/setup-team.js

```diff
@@ -24,15 +24,15 @@ const View = ({ psItem }) => {
           with the guidance of the NSC and UNEP
         </p>
       </div>
-      <div className={styles.tableSection}>
-        <h5 className={styles.title}>Team members</h5>
+      <div className="tableSection">
+        <h5 className="title">Team members</h5>
         <SetupTeamTable {...{ psItem, members, setMembers }} />
       </div>
-      <div className={styles.addMemberSection}>
-        <h5 className={styles.title}>Add a New Member</h5>
+      <div className="addMemberSection">
```

Comment:
Normally css classes use the `kebab-case` not camelCase

Reply:
We make an exception only for the root classes which are used by the styled component, because it is being attached in JS, which is camelCase